packages/vuetify/src/components/VDataTable/VDataTableHeaderMobile.ts

```diff
@@ -43,17 +43,14 @@ export default mixins(header).extend({
         },
       }, children)
     },
-    genSortSelect () {
-      const sortHeaders = this.headers.filter(h => h.sortable !== false && h.value !== 'data-table-select')
-
+    genSortSelect (items: Array<any>) {
```

Comment:
We use `any[]` syntax for array typings. It should also be `TableHeader[]`

Reply:
I'll change it. Do you also want me to use different commit msg and change the PR name not to include fix?

Reply:
It should go to dev, not master

Reply:
Changed